modules/ROOT/nav.adoc

```diff
@@ -21,26 +22,18 @@
   ** xref:upgrade-self-managed.adoc[Upgrade Runtime Fabric Using rtfctl]
   ** xref:upgrade-helm.adoc[Upgrade Runtime Fabric Using Helm]
   ** xref:upgrade-openshift.adoc[Upgrade Runtime Fabric as a Kubernetes (K8s) Operator]
+* xref:uninstall-self.adoc[Uninstall Runtime Fabric]
+* xref:rtf-k8s-practices.adoc[Kubernetes (K8s) Best Practices for Runtime Fabric]
+** xref:rtf-scale.adoc[Scalability Benchmarks]
+** xref:rtf-cluster-config.adoc[Cluster Configurations]
 * xref:manage-index.adoc[Manage Runtime Fabric]
  ** xref:view-health.adoc[View Runtime Fabric Health Details]
  ** xref:install-rtfctl.adoc[Use the Runtime Fabric Command Line Tool]
- ** xref:runtime-patch-updates.adoc[Manage Mule Runtime Patch Updates for Runtime Fabric]
  ** xref:replace-license-key.adoc[Replace a Mule License Key]
  ** xref:manage-backup-restore.adoc[Back Up and Restore Runtime Fabric]
- ** xref:enable-fips-140-2-compliance.adoc[Enable FIPS 140-2 Compliance Mode]
-* xref:uninstall-self.adoc[Uninstall Runtime Fabric]
-* xref:rtf-k8s-practices.adoc[Kubernetes (K8s) Best Practices for Runtime Fabric]
-** xref:rtf-scale.adoc[Scalability Benchmarks]
-** xref:rtf-cluster-config.adoc[Cluster Configurations]
-* xref:customize-kubernetes-crd.adoc[Customize Mule application Kubernetes Resources]
-* xref:authorized-namespaces.adoc[Configure Authorized Namespaces]
-* xref:deploy-resource-allocation-self-managed.adoc[Resource Allocation]
-* xref:migrate-helm.adoc[Migrating to helm from rtfctl]
-* xref:custom-ingress-configuration.adoc[Configure Ingress]
-* xref:configure-openshift-routes.adoc[Configure OpenShift Routes]
-* xref:persistence-gateway.adoc[Use Persistence Gateway]
-* xref:associate-environments.adoc[Associate Business Groups and Environments]
+ ** xref:migrate-helm.adoc[Migrating to helm from rtfctl]
```

Comment:
As this is a title, I think "Helm" should be capitalized.
Apart from that, if my guess is right, "rtfctl" is the acronym for "Runtime Fabric Command-Line Tool". 
If so, should it be capitalized and "RFCLT" (note that it should be "...CLT" instead of "...CTL").
Or is "rtfctl" something else? hahahaha